front/app/components/CustomFieldsForm/CustomFieldsPage.tsx

```diff
@@ -128,6 +135,8 @@ const CustomFieldsPage = ({
 
   return (
     <FormProvider {...methods}>
+      {currentPageNumber === lastPageNumber - 1 && <Feedback />}
```

Comment:
Only showing the Feedback component before submitting in case there are back-end errors. For the rest of the pages we rely on front-end validation. This is also the reason we need to scroll the errors into view with this form.